### DISCUSSION TITLE
EditContext with FluentValidation and FluentDatePicker

### DISCUSSION THREAD

**Discussion by nhwilly** [microsoft/fluentui-blazor#3030] [Q&A] [answered]
I'm trying to validate a date in my view model that is bound to my EditForm controls. 

I'm using BlazoredFluentValidation. 

My code is throwing as I discovered per the docs.  

What is the way to do this, then?

How do I validate it?  

Is there a workaround or has this been changed in 4.11x?

TIA

**Comment by dvoituron** [comment, marked as answer]
It's not very clear to me: we don't have and don't support a **BlazoredFluentValidation** component 😀

You can find a complete example of form validation at this address: https://www.fluentui-blazor.net/Forms
And the official Microsoft doc: https://learn.microsoft.com/en-us/aspnet/core/blazor/forms/validation

If you want to add a custom validation for the **Date** element, you can use an `OnValidSubmit=“ Submit”` method in which the validation can be adapted.

**Reply by nhwilly** [reply]
OK.  Fair enough.

Was trying to use: [this library](https://github.com/Blazored/FluentValidation) which works pretty well elsewhere.

Thanks for the quick response.

**Reply by nhwilly** [reply]
As it turns out, I was configuring it incorrectly.  It's working fine, now.  Apologize for the inconvenience.

**Reply by SeanFeldman** [reply]
@nhwilly, would you be able to share information/link to an example of using FluentUI for Blazor with FluentValidation library? I'm new to   FluentUI for Blazor and looking how that can be achieved.

**Reply by nhwilly** [reply]
The folks in charge of the project have decided to use another UI library.  I like FluentUi and I *really* like the guys supporting it, but I haven't looked at FluentUi in 6 months.  Sorry.  I think, if I remember correctly, you can just use it per the documentation from the library I referenced in the link.
